### DISCUSSION TITLE
about representation

### DISCUSSION THREAD

**Discussion by de-gozaru** [pyg-team/pytorch_geometric#5615] [Q&A]
Hi,

Is there a function in PyTorch geometric equivalent to `torch.bmm`  but with inputs in the PyG format, i.e. mat1 has the shape `Nxf`, where `N=bs*n`, and we have a `batch`  vector indicating for each entry, to which batch it belongs (and the same for mat2)?

Thank you in advance!

**Comment by rusty1s** [comment]
See `HeteroLinear`.
